### PR TITLE
Adding Pattern support for Lists

### PR DESCRIPTION
**Pull request by setchy**
#18

### CODE REVIEW

**Comment by setchy** [comment]
@bbakerman appreciate if you could review this PR and merge at your convenience

File: src/main/java/graphql/validation/constraints/standard/PatternConstraint.java [hunk 1]
```diff
@@ -44,28 +47,39 @@ public Documentation getDocumentation() {
...
         if (validatedValue == null) {
             return emptyList();
         }
-        String strValue = String.valueOf(validatedValue);
 
-        GraphQLDirective directive = validationEnvironment.getContextObject(GraphQLDirective.class);
+        List<Object> validatedValues;
+
+        if (isList(argumentType)) {
+            validatedValues = (ArrayList)validatedValue;
```
Comment (bbakerman) [inline src/main/java/graphql/validation/constraints/standard/PatternConstraint.java:65]: Cast as just `(List)` and not the specific type please

**Review by bbakerman — APPROVED** [review]
Thanks for this.  Just the one update and I am happy to accept it

File: src/main/java/graphql/validation/constraints/standard/PatternConstraint.java (hunk 1 above)
Comment (setchy) [reply src/main/java/graphql/validation/constraints/standard/PatternConstraint.java]: Certainly - update made

**Comment by setchy** [comment]
Thanks @bbakerman - should be all sorted now 😄

**Review by bbakerman — APPROVED** [review]

**Pull request merged**